scripts/python/missing_followers.py

```diff
@@ -0,0 +1,43 @@
+import json
+import sys
+
+if len(sys.argv) == 3:
+ try:
+    print "Old:", sys.argv[1]
+    print "New:", sys.argv[2]
+    json_file_old = open(sys.argv[1])
+    connections_old = json.load(json_file_old)
+    json_file_new = open(sys.argv[2])
+    connections_new = json.load(json_file_new)
+    follower_old = []
+    follower_new = []
+    for user in connections_old['followers']:
+        follower_old.append(user)
+    for user in connections_new['followers']:
+        follower_new.append(user)
+
+    found = None
```

Comment:
Let's keep the data types consistent and have `found` resolve to either `True` or `False`
Same with `overall`